nss-day-cohort-45/holiday-road-frozen-assets
# added some styling, added headings to dropdowns

## changes:
- added base styling, css resets
- added headings to dropdowns in index.html
- changed Select.js files to have += instead of = for the target innerHTML (to avoid overwriting headings)
- altered logo image to be more banner-like

## testing:
- fetch and serve this branch
- view index.html and see if it's cool with you
